cli/server/server.go

```diff
@@ -619,16 +622,20 @@ Main:
 // In case RPC or Prometheus or Pprof Address provided each of them will use it.
 // In case global Address (of the node) provided and RPC/Prometheus/Pprof don't have configured addresses they will
 // use global one. So Node and RPC and Prometheus and Pprof will run on one address.
+//
+// Deprecated: this method is leaved for backwards compatibility with deprecated
```

Comment:
left, but it's an internal one.

Reply:
Don't we need to mark internal APIs as deprecated? Removed.

Reply:
It's not exported, no one will see this deprecation notice.